diff -ru libressl-static/man/Makefile.am libressl-static-patched/man/Makefile.am
--- libressl-static/man/Makefile.am	2022-10-25 02:58:07.000000000 +0200
+++ libressl-static-patched/man/Makefile.am	2022-12-01 00:42:06.000000000 +0100
@@ -3367,14 +3367,6 @@
 	ln -sf "lh_new.3" "$(DESTDIR)$(mandir)/man3/LHASH_DOALL_ARG_FN_TYPE.3"
 	ln -sf "lh_new.3" "$(DESTDIR)$(mandir)/man3/LHASH_DOALL_FN_TYPE.3"
 	ln -sf "lh_new.3" "$(DESTDIR)$(mandir)/man3/LHASH_HASH_FN_TYPE.3"
-	ln -sf "lh_new.3" "$(DESTDIR)$(mandir)/man3/lh_<type>_delete.3"
-	ln -sf "lh_new.3" "$(DESTDIR)$(mandir)/man3/lh_<type>_doall.3"
-	ln -sf "lh_new.3" "$(DESTDIR)$(mandir)/man3/lh_<type>_doall_arg.3"
-	ln -sf "lh_new.3" "$(DESTDIR)$(mandir)/man3/lh_<type>_error.3"
-	ln -sf "lh_new.3" "$(DESTDIR)$(mandir)/man3/lh_<type>_free.3"
-	ln -sf "lh_new.3" "$(DESTDIR)$(mandir)/man3/lh_<type>_insert.3"
-	ln -sf "lh_new.3" "$(DESTDIR)$(mandir)/man3/lh_<type>_new.3"
-	ln -sf "lh_new.3" "$(DESTDIR)$(mandir)/man3/lh_<type>_retrieve.3"
 	ln -sf "lh_new.3" "$(DESTDIR)$(mandir)/man3/lh_delete.3"
 	ln -sf "lh_new.3" "$(DESTDIR)$(mandir)/man3/lh_doall.3"
 	ln -sf "lh_new.3" "$(DESTDIR)$(mandir)/man3/lh_doall_arg.3"
@@ -6276,14 +6268,6 @@
 	-rm -f "$(DESTDIR)$(mandir)/man3/LHASH_DOALL_ARG_FN_TYPE.3"
 	-rm -f "$(DESTDIR)$(mandir)/man3/LHASH_DOALL_FN_TYPE.3"
 	-rm -f "$(DESTDIR)$(mandir)/man3/LHASH_HASH_FN_TYPE.3"
-	-rm -f "$(DESTDIR)$(mandir)/man3/lh_<type>_delete.3"
-	-rm -f "$(DESTDIR)$(mandir)/man3/lh_<type>_doall.3"
-	-rm -f "$(DESTDIR)$(mandir)/man3/lh_<type>_doall_arg.3"
-	-rm -f "$(DESTDIR)$(mandir)/man3/lh_<type>_error.3"
-	-rm -f "$(DESTDIR)$(mandir)/man3/lh_<type>_free.3"
-	-rm -f "$(DESTDIR)$(mandir)/man3/lh_<type>_insert.3"
-	-rm -f "$(DESTDIR)$(mandir)/man3/lh_<type>_new.3"
-	-rm -f "$(DESTDIR)$(mandir)/man3/lh_<type>_retrieve.3"
 	-rm -f "$(DESTDIR)$(mandir)/man3/lh_delete.3"
 	-rm -f "$(DESTDIR)$(mandir)/man3/lh_doall.3"
 	-rm -f "$(DESTDIR)$(mandir)/man3/lh_doall_arg.3"
Only in libressl-static-patched/man: Makefile.am.orig
diff -ru libressl-static/man/Makefile.in libressl-static-patched/man/Makefile.in
--- libressl-static/man/Makefile.in	2022-10-25 02:58:21.000000000 +0200
+++ libressl-static-patched/man/Makefile.in	2022-12-01 00:42:06.000000000 +0100
@@ -915,7 +915,7 @@
 
 install-data-am: install-man
 	@$(NORMAL_INSTALL)
-	$(MAKE) $(AM_MAKEFLAGS) install-data-hook
+
 install-dvi: install-dvi-am
 
 install-dvi-am:
@@ -3780,14 +3780,6 @@
 @ENABLE_LIBTLS_ONLY_FALSE@	ln -sf "lh_new.3" "$(DESTDIR)$(mandir)/man3/LHASH_DOALL_ARG_FN_TYPE.3"
 @ENABLE_LIBTLS_ONLY_FALSE@	ln -sf "lh_new.3" "$(DESTDIR)$(mandir)/man3/LHASH_DOALL_FN_TYPE.3"
 @ENABLE_LIBTLS_ONLY_FALSE@	ln -sf "lh_new.3" "$(DESTDIR)$(mandir)/man3/LHASH_HASH_FN_TYPE.3"
-@ENABLE_LIBTLS_ONLY_FALSE@	ln -sf "lh_new.3" "$(DESTDIR)$(mandir)/man3/lh_<type>_delete.3"
-@ENABLE_LIBTLS_ONLY_FALSE@	ln -sf "lh_new.3" "$(DESTDIR)$(mandir)/man3/lh_<type>_doall.3"
-@ENABLE_LIBTLS_ONLY_FALSE@	ln -sf "lh_new.3" "$(DESTDIR)$(mandir)/man3/lh_<type>_doall_arg.3"
-@ENABLE_LIBTLS_ONLY_FALSE@	ln -sf "lh_new.3" "$(DESTDIR)$(mandir)/man3/lh_<type>_error.3"
-@ENABLE_LIBTLS_ONLY_FALSE@	ln -sf "lh_new.3" "$(DESTDIR)$(mandir)/man3/lh_<type>_free.3"
-@ENABLE_LIBTLS_ONLY_FALSE@	ln -sf "lh_new.3" "$(DESTDIR)$(mandir)/man3/lh_<type>_insert.3"
-@ENABLE_LIBTLS_ONLY_FALSE@	ln -sf "lh_new.3" "$(DESTDIR)$(mandir)/man3/lh_<type>_new.3"
-@ENABLE_LIBTLS_ONLY_FALSE@	ln -sf "lh_new.3" "$(DESTDIR)$(mandir)/man3/lh_<type>_retrieve.3"
 @ENABLE_LIBTLS_ONLY_FALSE@	ln -sf "lh_new.3" "$(DESTDIR)$(mandir)/man3/lh_delete.3"
 @ENABLE_LIBTLS_ONLY_FALSE@	ln -sf "lh_new.3" "$(DESTDIR)$(mandir)/man3/lh_doall.3"
 @ENABLE_LIBTLS_ONLY_FALSE@	ln -sf "lh_new.3" "$(DESTDIR)$(mandir)/man3/lh_doall_arg.3"
@@ -6689,14 +6681,6 @@
 @ENABLE_LIBTLS_ONLY_FALSE@	-rm -f "$(DESTDIR)$(mandir)/man3/LHASH_DOALL_ARG_FN_TYPE.3"
 @ENABLE_LIBTLS_ONLY_FALSE@	-rm -f "$(DESTDIR)$(mandir)/man3/LHASH_DOALL_FN_TYPE.3"
 @ENABLE_LIBTLS_ONLY_FALSE@	-rm -f "$(DESTDIR)$(mandir)/man3/LHASH_HASH_FN_TYPE.3"
-@ENABLE_LIBTLS_ONLY_FALSE@	-rm -f "$(DESTDIR)$(mandir)/man3/lh_<type>_delete.3"
-@ENABLE_LIBTLS_ONLY_FALSE@	-rm -f "$(DESTDIR)$(mandir)/man3/lh_<type>_doall.3"
-@ENABLE_LIBTLS_ONLY_FALSE@	-rm -f "$(DESTDIR)$(mandir)/man3/lh_<type>_doall_arg.3"
-@ENABLE_LIBTLS_ONLY_FALSE@	-rm -f "$(DESTDIR)$(mandir)/man3/lh_<type>_error.3"
-@ENABLE_LIBTLS_ONLY_FALSE@	-rm -f "$(DESTDIR)$(mandir)/man3/lh_<type>_free.3"
-@ENABLE_LIBTLS_ONLY_FALSE@	-rm -f "$(DESTDIR)$(mandir)/man3/lh_<type>_insert.3"
-@ENABLE_LIBTLS_ONLY_FALSE@	-rm -f "$(DESTDIR)$(mandir)/man3/lh_<type>_new.3"
-@ENABLE_LIBTLS_ONLY_FALSE@	-rm -f "$(DESTDIR)$(mandir)/man3/lh_<type>_retrieve.3"
 @ENABLE_LIBTLS_ONLY_FALSE@	-rm -f "$(DESTDIR)$(mandir)/man3/lh_delete.3"
 @ENABLE_LIBTLS_ONLY_FALSE@	-rm -f "$(DESTDIR)$(mandir)/man3/lh_doall.3"
 @ENABLE_LIBTLS_ONLY_FALSE@	-rm -f "$(DESTDIR)$(mandir)/man3/lh_doall_arg.3"
Only in libressl-static-patched/man: Makefile.in.orig
