diff -ru rl_json0.9.12-orig//generic/rl_json.decls rl_json0.9.12/generic/rl_json.decls
--- rl_json0.9.12-orig//generic/rl_json.decls	2018-08-14 15:47:55 +0000
+++ rl_json0.9.12/generic/rl_json.decls	2019-02-08 15:11:57 +0000
@@ -1,5 +1,7 @@
 library rl_json
 interface rl_json
+scspec extern
+
 declare 0 generic {
 	Tcl_Obj *JSON_NewJvalObj(int type, Tcl_Obj *val)
 }
diff -ru rl_json0.9.12-orig//generic/rl_jsonDecls.h rl_json0.9.12/generic/rl_jsonDecls.h
--- rl_json0.9.12-orig//generic/rl_jsonDecls.h	2018-08-14 15:47:55 +0000
+++ rl_json0.9.12/generic/rl_jsonDecls.h	2019-02-08 15:12:15 +0000
@@ -10,16 +10,16 @@
  */
 
 /* 0 */
-EXTERN Tcl_Obj *	JSON_NewJvalObj(int type, Tcl_Obj *val);
+extern Tcl_Obj *	JSON_NewJvalObj(int type, Tcl_Obj *val);
 /* 1 */
-EXTERN int		JSON_GetJvalFromObj(Tcl_Interp *interp, Tcl_Obj *obj,
+extern int		JSON_GetJvalFromObj(Tcl_Interp *interp, Tcl_Obj *obj,
 				int *type, Tcl_Obj **val);
 /* 2 */
-EXTERN int		JSON_Set(Tcl_Interp*interp, Tcl_Obj*srcvar,
+extern int		JSON_Set(Tcl_Interp*interp, Tcl_Obj*srcvar,
 				Tcl_Obj *const pathv[], int pathc,
 				Tcl_Obj*replacement);
 /* 3 */
-EXTERN int		JSON_Template(Tcl_Interp*interp, Tcl_Obj*template,
+extern int		JSON_Template(Tcl_Interp*interp, Tcl_Obj*template,
 				Tcl_Obj*dict, Tcl_Obj**res);
 
 typedef struct Rl_jsonStubs {
