diff -ru tclcurl/Makefile.in tclcurl-patched/Makefile.in
--- tclcurl/Makefile.in	2021-03-18 18:28:21.000000000 +0100
+++ tclcurl-patched/Makefile.in	2022-04-26 16:07:48.383456736 +0200
@@ -395,7 +395,7 @@
 	done
 	@if test "x$(SHARED_BUILD)" = "x1"; then \
 	    echo " Install pkgIndex.tcl $(DESTDIR)$(pkglibdir)"; \
-	    $(INSTALL_DATA) $(srcdir)/pkgIndex.tcl $(DESTDIR)$(pkglibdir); \
+	    $(INSTALL_DATA) pkgIndex.tcl $(DESTDIR)$(pkglibdir); \
 	fi
 
 #========================================================================
