diff -ru Tktable2.10/generic/tkTable.c newTktable2.10/generic/tkTable.c
--- Tktable2.10/generic/tkTable.c	2008-11-14 17:43:35.000000000 -0600
+++ newTktable2.10/generic/tkTable.c	2020-08-29 10:44:15.000000000 -0500
@@ -1680,7 +1680,7 @@
  * rectangles.  Thus turn it off for all cases until clip rectangles
  * are known to be respected. [Bug 1805350]
  */
-#if 1 || defined(MAC_TCL) || defined(UNDER_CE) || (defined(WIN32) && defined(TCL_THREADS)) || defined(MAC_OSX_TK)
+#if !defined(MAC_OSX_TK)
 #define NO_XSETCLIP
 #endif
 /*
diff -ru Tktable2.10/generic/tkTable.h newTktable2.10/generic/tkTable.h
--- Tktable2.10/generic/tkTable.h	2005-02-01 22:22:25.000000000 -0600
+++ newTktable2.10/generic/tkTable.h	2020-08-29 10:48:46.000000000 -0500
@@ -15,6 +15,13 @@
 #ifndef _TKTABLE_H_
 #define _TKTABLE_H_
 
+#ifdef MAC_OSX_TK
+ #ifndef MAC_OSX_TCL
+ #define MAC_OSX_TCL
+ #endif
+#endif
+#define panic Tcl_Panic
+
 #include <string.h>
 #include <stdlib.h>
 #include <ctype.h>
