diff -ru treectrl2.4.2/configure treectrl2.4.2-patched/configure
--- treectrl2.4.2/configure	2018-02-16 15:24:34.000000000 +0100
+++ treectrl2.4.2-patched/configure	2018-12-16 00:58:33.000000000 +0100
@@ -6427,7 +6427,7 @@
 	    { $as_echo "$as_me:${as_lineno-$LINENO}: result: Cocoa" >&5
 $as_echo "Cocoa" >&6; }
 
-    PKG_CFLAGS="$PKG_CFLAGS -DMAC_TK_COCOA -std=gnu99 -x objective-c -fobjc-gc"
+    PKG_CFLAGS="$PKG_CFLAGS -DMAC_TK_COCOA -std=gnu99 -x objective-c"
 
 
 
diff -ru treectrl2.4.2/configure.ac treectrl2.4.2-patched/configure.ac
--- treectrl2.4.2/configure.ac	2018-02-16 15:24:34.000000000 +0100
+++ treectrl2.4.2-patched/configure.ac	2018-12-16 00:59:02.000000000 +0100
@@ -177,7 +177,7 @@
     case ${TK_LIBS} in
 	*Cocoa*)
 	    AC_MSG_RESULT([Cocoa])
-	    TEA_ADD_CFLAGS([-DMAC_TK_COCOA -std=gnu99 -x objective-c -fobjc-gc])
+	    TEA_ADD_CFLAGS([-DMAC_TK_COCOA -std=gnu99 -x objective-c])
 	    TEA_ADD_SOURCES([tkMacOSXTree.c])
 	    TEA_ADD_LIBS([-framework Cocoa -framework Carbon])
 	    ;;
